wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests+OneOnOneConversation.swift

```diff
@@ -37,13 +37,6 @@ class DatabaseMigrationTests_OneOnOneConversation: XCTestCase {
     }
 
     func testMigratingToMessagingStore_2_114_UpdatesRelationships() throws {
```

Comment:
Should the name change to something like `testMigratingToMessagingStore_from2_113_updatesRelationships()`?